### PR TITLE
fix chisel windows prompt

### PR DESCRIPTION
**Pull request by daog1**
https://github.com/foundry-rs/foundry/issues/4503

### CODE REVIEW

File: chisel/src/bin/chisel.rs [hunk 1]
```diff
@@ -134,7 +134,7 @@ async fn main() -> eyre::Result<()> {
     loop {
         // Get the prompt from the dispatcher
         // Variable based on status of the last entry
-        let prompt = dispatcher.get_prompt();
+        let prompt = if cfg!(windows) { format!(">") }else {dispatcher.get_prompt()};
```
Comment (mattsse) [inline chisel/src/bin/chisel.rs:137]: the prompt is also used in the eval loop:

https://github.com/foundry-rs/foundry/blob/master/chisel/src/bin/chisel.rs#L137-L137

is this change sufficient or should the cfg check be moved to the `get_prompt` function instead?

tbh, not familiar with why we need to adjust this for windows.

Comment (daog1) [reply chisel/src/bin/chisel.rs:137]: That's the easiest way to change it.
The Windows command line does not support this complex command line prompt, and the Windows input cursor is not in the right position after the prompt.

Comment (DaniPopes) [reply chisel/src/bin/chisel.rs:137]: evcxr (rust repl in rust) does it like this:

https://github.com/evcxr/evcxr/blob/b7f5f9b80bb77b77eb9e04eec55a6a61cf4b2862/evcxr_repl/src/bin/evcxr.rs#L259-L260

Comment (mattsse) [reply chisel/src/bin/chisel.rs:137]:
```
 #[cfg(windows)]
 colored::control::set_virtual_terminal(true).ok();
```    
this looks like the appropriate fix @DaniPopes ?

Comment (daog1) [reply chisel/src/bin/chisel.rs:137]: I will test his writing method later

**Pull request closed without merging**